src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  padding: 10px;
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: calc(7 * ($cell-size + $gap));
+
+  @each $start-day, $margin-left in $weekdays {
```

Comment:
The variable `$weekdays` is used here but is not defined within this file. Ensure that `$weekdays` is defined either in this file or imported from another SCSS file.